dc-infrastructure.tf

```diff
@@ -194,4 +194,6 @@ module "bitbucket" {
   elasticsearch_mem      = var.bitbucket_elasticsearch_mem
   elasticsearch_storage  = var.bitbucket_elasticsearch_storage
   elasticsearch_replicas = var.bitbucket_elasticsearch_replicas
+
+  shared_home_snapshot_id = var.bitbucket_shared_home_snapshot_id
```

Comment:
This variable will need to be added to all products (`confluence_shared_home_snapshot_id`, `jira_shared_home_snapshot_id`)